usb-to-can_socketcan/ixx_usb_fd.c

```diff
@@ -614,20 +614,20 @@ static int ixx_usbfd_handle_canmsg(struct ixx_usb_device *dev,
 	if (!skb)
 		return -ENOMEM;
 
-	if (flags & IXXAT_USBFD_MSG_FLAGS_EDL) {
+
+    	if (flags & IXXAT_USBFD_MSG_FLAGS_EDL) {
 		if (flags & IXXAT_USBFD_MSG_FLAGS_FDR)
 			can_frame->flags |= CANFD_BRS;
 
 		if (flags & IXXAT_USBFD_MSG_FLAGS_ESI)
 			can_frame->flags |= CANFD_ESI;
 
 		can_frame->len =
-			can_dlc2len(
-			get_canfd_dlc((flags & IXXAT_USBFD_MSG_FLAGS_DLC)
-			>> 16));
+			can_fd_dlc2len((flags & IXXAT_USBFD_MSG_FLAGS_DLC)
+			>> 16);
```

Comment:
As per https://www.spinics.net/lists/linux-can/msg05423.html it seems socketcan simply renamed can_dlc2len to can_fd_dlc2len, unless i'm reading wrong. 

This is also related https://www.spinics.net/lists/linux-can/msg05440.html